### PR TITLE
test: fix Tab widget related cypress tests

### PR DESCRIPTION
**Pull request by Parthvi12**
## Description

Contains fix for test Bind_TabWidget_Input_spec and Tab_new_scenario_spec

## How Has This Been Tested?

locally

## Checklist:

- [] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:white_circle: Total coverage has not changed</summary>
No changes to code coverage between the base branch and the head branch</details>

### CODE REVIEW

**Review by Aishwarya-U-R — APPROVED** [review]

File: app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/LayoutWidgets/Tab_new_scenario_spec.js [hunk 1]
```diff
@@ -20,6 +20,8 @@ describe("Tab widget test", function() {
     cy.get(publish.buttonWidget).should("be.visible");
     cy.get(publish.textWidget).should("be.visible");
     cy.get(publish.datePickerNew).should("be.visible");
+    cy.wait(3000);
+    // cy.get('.t--widget-iconbuttonwidget').click({force:true})
```
Comment (keyurparalkar) [inline app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/LayoutWidgets/Tab_new_scenario_spec.js:24]: @Parthvi12 Can you remove this commented code?

File: app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/Binding/Bind_TabWidget_Input_spec.js [hunk 2]
```diff
@@ -27,17 +27,19 @@ describe("Binding the input Widget with tab Widget", function() {
...
       .contains("Tab 2")
       .click({ force: true })
+      .wait(2000)
       .should("have.class", "is-selected");
 
     cy.get(publish.inputWidget + " " + "input")
       .first()
       .invoke("attr", "value")
       .should("contain", "Tab 2");
+
```
Comment (keyurparalkar) [inline app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/Binding/Bind_TabWidget_Input_spec.js:37]: Remove this blank line.

**Review by keyurparalkar — CHANGES_REQUESTED** [review]

**Pull request merged**